src/plugins/explore/public/components/query_panel/components/editor_stack/types.ts

```diff
@@ -9,7 +9,7 @@ export enum EditorType {
 }
 
 export enum LanguageType {
-  Natural = 'natural',
-  KeyValue = 'keyvalue',
-  PPL = 'ppl',
+  Natural = 'plaintext',
+  KeyValue = 'PPL',
+  PPL = 'PPL',
```

Comment:
we have a constant for default language for explore. we can use that constant instead of hard coding.